### ISSUE TITLE
Move appveyor to intake-dev

### ISSUE THREAD

**Issue by jsignell** [ContinuumIO/intake#260]
I realized that the appveyor setup is under my name. We should more it to intake-dev. Is there an associated email account or should there be? Maybe @bryevdv has advice on setting this kind of stuff up.

**Comment by bryevdv** [comment]
We have a *bokeh-integrations* GitHub account that we use for external GH apps, that is tied to a *bokehplots* Gmail account. Credentials are stored in the team 1Pass

**Comment by jsignell** [comment]
Ok thanks that is good info.

**Comment by martindurant** [comment]
Is there anything to be done here?

**Comment by jsignell** [comment]
I think so yeah

**Comment by jsignell** [comment]
<img width="721" alt="Screen Shot 2019-05-22 at 4 48 18 PM" src="https://user-images.githubusercontent.com/4806877/58207848-7f66a600-7cb1-11e9-8a7e-0eb2d45b5297.png">

**Comment by martindurant** [comment]
I don't know... ask other OSS people?

**Comment by bryevdv** [comment]
I made a service account `service@bokeh.org` to use for things like this (and `service-backup@bokeh.org` recovery account) If you don't have your own domain/gsuite then `intake-service@gmail.com`, etc. is fine (we used a gmail service account for many years)

**Comment by jsignell** [comment]
Yeah we have `intake.dev@gmail.com`. I posted on appveyor support back in February and they didn't respond. It seems that appveyor isn't running on my own account either though which is what I thought was setup.
